geomstats/geometry/discrete_curves.py

```diff
@@ -1905,3 +1892,26 @@ def align(self, point, base_point):
                 for point_, base_point_ in zip(point, base_point)
             ]
         )
+
+
+register_quotient_structure(
```

Comment:
NIT: See my comment on the PR on alternate aligner about naming the parameters for readbility.